File: AppointmentSchedulerAPI/layers/ServiceLayer/v1/Controllers/DTO/Response/SystemNotificationDTO.cs
using AppointmentSchedulerAPI.layers.BusinessLogicLayer.Model.Types.Notification;

namespace AppointmentSchedulerAPI.layers.ServiceLayer.v1.Controllers.DTO.Response
{
    public class SystemNotificationDTO : NotificationDTO
    {
        public required SystemNotificationCodeType Code { get; set;}
        public required SystemNotificationSeverityCodeType Severity { get; set;}

    }
}

File: AppointmentSchedulerWeb/src/model/dtos/system-notification.dto.ts
import { SystemNotificationCodeType } from "../../view-model/business-entities/types/system-notification-code.types";
import { SystemNotificationSeverityCode } from "../../view-model/business-entities/types/system-notification-severity-code.types";
import { NotificationDTO } from "./notification.dto";

export interface SystemNotificationDTO extends NotificationDTO {
  code: SystemNotificationCodeType,
  severity: SystemNotificationSeverityCode
}


